src/components/VideoPlayerContexts/VideoPopoverMenuContext.tsx

```diff
@@ -24,9 +24,9 @@ function VideoPopoverMenuContextProvider({children}: ChildrenProps) {
     const updatePlaybackSpeed = useCallback(
         (speed: PlaybackSpeed) => {
             setCurrentPlaybackSpeed(speed);
-            currentVideoPlayerRef.current?.setStatusAsync?.({rate: speed});
+            videoPopoverMenuPlayerRef.current?.setStatusAsync?.({rate: speed});
```

Comment:
There was [regression](https://github.com/Expensify/App/issues/42519) from this change.
More details: https://github.com/Expensify/App/issues/42519#issuecomment-2143431227